src/test/kotlin/nl/info/client/zgw/model/ZrcFixtures.kt

```diff
@@ -111,12 +111,12 @@ fun createRolMedewerker(
 )
 
 fun createRolNatuurlijkPersoon(
-    zaaktypeURI: URI = URI("https://example.com/${UUID.randomUUID()}"),
-    rolType: RolType = createRolType(zaakTypeUri = zaaktypeURI),
+    zaakURI: URI = URI("https://example.com/${UUID.randomUUID()}"),
```

Comment:
This is the URI reference to a zaak type right? Why name it `zaakURI` ?

Reply:
No, zaak URI. See above.

Reply:
A rol is linked to a zaak, it has nothing to do with a zaaktype.